### PR TITLE
Fix:  New TPDB domain

### LINKED ISSUE #161: tpdb has been shutdown
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

the https://metadataapi.net/ cannot be access

there is the announce

Site Shutdown Announcement
We regret to inform you that our site is being shut down. We appreciate your support and engagement throughout the years.

However, we have some good news! We will be publishing our dataset soon, so stay tuned for updates.

### Expected Behavior

normal access

### Steps To Reproduce

_No response_

### Environment

```markdown
android
```

### What branch are you running?

Main

### Trace Logs?

no log

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by plz12345**
#### Database Migration
NO

#### Description
Simple replace from old domain to new one.  Tested the GraphQL API via Stash to confirm it's operational.

#### Todos
- [ x] Tests
- [x ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [n/a ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #161

### CODE REVIEW

**Review by gothdawid — COMMENTED** [review]
looks good

**Comment by hummelm10** [comment]
This might be bigger than just a find and replace, the site appears to redirect to a login screen which I don't believe the old one did.

**Comment by plz12345** [comment]
I'm pretty sure all the "failed" unit tests are failing for all PR's,, not just mine.  Most of the ones I reviewed were against the core .NET code, which I didn't touch.  The lint-related ones are likely faiing due to being inconsistent with the build.sh script's checks, which I do run before pushing a PR.

**Review by Bananenbaas — APPROVED** [review]
Great, lets go!

**Comment by plz12345** [comment]
> Great, lets go!

Can you merge?

**Comment by Bananenbaas** [comment]
> > Great, lets go!
> 
> Can you merge?

Nah I cannot merge :(

**Review by pirgah69 — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -2305,15 +2305,10 @@ camelcase@^5.3.1:
...
-caniuse-lite@^1.0.30001464, caniuse-lite@^1.0.30001489:
-  version "1.0.30001494"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001494.tgz#3e56e04a48da7a79eae994559eb1ec02aaac862f"
-  integrity sha512-sY2B5Qyl46ZzfYDegrl8GBCzdawSLT4ThM9b9F+aDYUrAG2zCOyMbd2Tq34mS1g4ZKBfjRlzOohQMxx28x6wJg==
-
-caniuse-lite@^1.0.30001541:
-  version "1.0.30001551"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001551.tgz#1f2cfa8820bd97c971a57349d7fd8f6e08664a3e"
-  integrity sha512-vtBAez47BoGMMzlbYhfXrMV1kvRF2WP/lqiMuDu1Sb4EE4LKEgjopFDSRtZfdVnslNRpOqV/woE+Xgrwj6VQlg==
+caniuse-lite@^1.0.30001464, caniuse-lite@^1.0.30001489, caniuse-lite@^1.0.30001541:
```
Comment (Qstick) [inline yarn.lock]: Why the yarn.lock file change with this?

**Review by Qstick — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (plz12345) [reply yarn.lock]: When running `./build.sh --lint`, it nags about running `npx update-browserslist-db@latest` often.  I ran it and it updated that lock file, so I included it in the commit.

Comment (Qstick) [reply yarn.lock]: Don't include it, it's best for a separate contribution. It's unrelated to this change

Comment (plz12345) [reply yarn.lock]: OK, no worries.  Updated the commit to not include the lock file.

**Review by Qstick — APPROVED** [review]

**Pull request merged**